docs/docs/docs/getting-started/installation.md

```
@@ -196,7 +196,66 @@ Click on the image below to play the video.
+   cp envFiles/.env.devcontainer .env
+   ```
+1. Change the `.env` file's `API_POSTGRES_HOST` parameter to `localhost`.
+   ```ini
+   API_POSTGRES_HOST=localhost
+   ```
+1. Install the packages
+   ```bash
+   pnpm install
+   ```
+1. You will now need to make your user a part of the `docker` operating system group or else you will get `permission denied` messages when starting docker later. `$USER` is a universal representation of your username. You don't need to change this in the command below.
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)